File: app/src/main/java/com/returnnotfound/stain/base/utils/DrawableUtils.java
package com.returnnotfound.stain.base.utils;

import android.graphics.Color;
import android.graphics.drawable.Drawable;
import android.graphics.drawable.GradientDrawable;

public class DrawableUtils {
  public static Drawable getDrawable(int radius, String color) {
    GradientDrawable gradientDrawable = new GradientDrawable();
    gradientDrawable.setCornerRadius(radius);
    try {
      gradientDrawable.setColor(Color.parseColor(color));
    } catch (Exception e) {
      gradientDrawable.setColor(Color.parseColor("#de4242"));
    }
    return gradientDrawable;
  }
}

File: app/src/main/java/com/returnnotfound/stain/base/utils/DrawableUtils.kt
package com.returnnotfound.stain.base.utils

import android.graphics.Color
import android.graphics.drawable.Drawable
import android.graphics.drawable.GradientDrawable

object DrawableUtils {
  @JvmStatic
  fun getDrawable(radius: Int, color: String?): Drawable {
    val gradientDrawable = GradientDrawable()
    gradientDrawable.cornerRadius = radius.toFloat()
    try {
      gradientDrawable.setColor(Color.parseColor(color))
    } catch (e: Exception) {
      gradientDrawable.setColor(Color.parseColor("#de4242"))
    }
    return gradientDrawable
  }
}